corehq/apps/domain/tests/test_password_strength.py

```diff
@@ -34,6 +31,26 @@ def test_sensitivity_to_minimum_zxcvbn_score_setting_bad(self):
     def test_sensitivity_to_minimum_zxcvbn_score_setting_good(self):
         self.assert_good_password(PASSWORDS_BY_STRENGTH[3])
 
+    # Test minimum password length
+    @override_settings(MINIMUM_PASSWORD_LENGTH=8, MINIMUM_ZXCVBN_SCORE=0)
+    def test_length_5_password(self):
+        self.assert_bad_password("e3r4f")
+
+    # Password has less than the minimum requirement
+    @override_settings(MINIMUM_PASSWORD_LENGTH=7)
+    def test_score_1_length_9_password(self):
+        self.assert_bad_password(PASSWORDS_BY_STRENGTH[1])
+
+    # Password has exactly the minimum requirement
+    @override_settings(MINIMUM_PASSWORD_LENGTH=7)
+    def test_score_2_length_7_password(self):
+        self.assert_good_password(PASSWORDS_BY_STRENGTH[2])
+
+    # Password has more than the minimum requirement
+    @override_settings(MINIMUM_SUBSCRIPTION_LENGTH=8)
+    def test_score_3_length_10_password(self):
+        self.assert_good_password(PASSWORDS_BY_STRENGTH[3])
+
```

Comment:
If I change my test names should I also change the other test names in this file? For example, naming `test_score_1_password` from other authors as `test_weak_password_is_rejected`. ( I have a concern, because I don't know if other author will be happy with that)
Also, if I named `test_score_1_length_9_password` as you suggested, there would be a name collision, because `test_length_5_password` can also be named as `test_too_short_password_is_rejected`.
The name collision could be resolved, but that would result in very long test names, I'm wondering how long of a test name is acceptable?
Could you clarify on that?